tests/routes_tests/cbauthentication_tests/cb_login.test.js

```diff
@@ -0,0 +1,35 @@
+const dbConnection = require('../../../react-backend/database/dbConnection.js');
+const test = require('tape');
+const request = require('supertest');
+const app = require('../../../react-backend/app');
+
+test('Route cblogin | post | successful request', (t) => {
+  t.plan(3);
+  const successPayload = { formEmail: 'findmyfroggy@frogfinders.com', formPswd: 'Funnyfingers11!' };
+  const failPayload = { formEmail: 'findmyfroggy@frogfinders.com', formPswd: 'password123' };
+  const emptyPayload = { formEmail: '', formPswd: '' };
+  request(app)
+    .post('/api/cb/login')
+    .send(successPayload)
+    .expect(200)
+    .expect('Content-Type', /json/)
+    .end((err, res) => {
+      t.ok(res.body.success, 'cb successfully logged in with correct password');
+    });
+  request(app)
+    .post('/api/cb/login')
+    .send(failPayload)
+    .expect(200)
```

Comment:
It doesn't seem right that a request where authentication is rejected returns a `200`. `401` is the code I would expect. Why is this the case?

Reply:
Looks like the server loves dishing out `200`'s. I'll change files individually as I write route tests.

Reply:
Be careful, the client app may rely on the fact that 200 codes are being returned 😕